### PR TITLE
[Catalog] Change settings to deploy to device

### PR DESCRIPTION
**Pull request by randallli**
I have to constantly change the settings in the catalog project in order to deploy to device.

This change commits those changes that are required every time a Googler deploys to device.

### CODE REVIEW

File: catalog/MDCCatalog.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -333,6 +333,7 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
+				English,
```
Comment (bryanoltman) [inline catalog/MDCCatalog.xcodeproj/project.pbxproj]: This seems unrelated

File: catalog/MDCCatalog.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings [hunk 2]
```diff
@@ -4,5 +4,7 @@
 <dict>
 	<key>BuildSystemType</key>
 	<string>Original</string>
+	<key>PreviewsEnabled</key>
```
Comment (bryanoltman) [inline catalog/MDCCatalog.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings]: Same as above

**Review by bryanoltman — APPROVED** [review]

File: catalog/MDCCatalog.xcodeproj/project.pbxproj (hunk 1 above)
Comment (randallli) [reply catalog/MDCCatalog.xcodeproj/project.pbxproj]: Done.

File: catalog/MDCCatalog.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings (hunk 2 above)
Comment (randallli) [reply catalog/MDCCatalog.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings]: Done.
